docs/content/direct/start-from-ocm.md

```diff
@@ -0,0 +1,143 @@
+produce a very simple system --- suitable for study but not production
+usage. For general setup information, see [the full
+story](user-guide-intro.md#the-full-story).
+
+### Install software prerequisites
+
+The following command will check for the prerequisites that you will need for the later steps. See [the prerequisites doc](pre-reqs.md) for more details.
+
+```shell
+bash <(curl https://raw.githubusercontent.com/kubestellar/kubestellar/v{{ config.ks_latest_regular_release }}/hack/check_pre_req.sh) kflex ocm helm kubectl docker kind
+```
+
+### Cleanup from previous runs
+
+If you have run this recipe or used `kflex` (the KubeFlex command
+line) in any other capacity, it may have some troublesome hidden state
+in your kubeconfig file. The following command clears out that hidden
+state.
+
+```shell
+yq -i 'del(.preferences)' ${KUBECONFIG:-$HOME/.kube/config}
+```
+
+### Set the Version appropriately as an environment variable
+
+```shell
+export KUBESTELLAR_VERSION={{ config.ks_latest_release }}
+```
+
+### OCM Quick Start with Ingress
+
+This recipe uses a modified version of [the OCM Quick Start](https://raw.githubusercontent.com/open-cluster-management-io/OCM/main/solutions/setup-dev-environment/local-up.sh). The modification is because KubeStellar needs the hosting cluster to have an Ingress controller with SSL passthrough enabled. The modified Quick Start script has the following modifications compared to the baseline.
+
+1. The `kind` cluster created for the hub has an additional port mapping, where the Ingress controller listens.
+1. The script installs an Ingress controller (based on nginx) into the hub cluster, then patches it to enable SSL passthrough, and later waits for it to be in service.
+
+You can invoke the modified OCM Quick Start as follows.
+
+```shell
+curl -L https://raw.githubusercontent.com/kubestellar/kubestellar/refs/heads/main/scripts/ocm-local-up-for-ingress.sh | bash
+```
+
+Like the baseline, this script creates a `kind` cluster named "hub" to
+serve as hub cluster (known in KubeStellar as an Inventory and
+Transport Space, ITS) and two `kind` clusters named "cluster1" and
+"cluster2" to serve as managed clusters (known in KubeStellar as
+Workload Execution Clusters, WECs), and registers them in the hub.
+
+### Label WECs for selection by examples
+
+The examples will use label selectors to direct workload to WECs (in
+terms of their ManagedCluster representations). The following commands
+apply labels that will be used in the examples.
+
+```shell
+kubectl --context kind-hub label managedcluster cluster1 location-group=edge name=cluster1
+kubectl --context kind-hub label managedcluster cluster2 location-group=edge name=cluster2
+```
+
+### Use Core Helm chart to initialize KubeFlex, recognize ITS, and create WDS
+
+This chart instance will do the following.
+
+- Install KubeFlex in the hosting cluster.
+- Tell that the hosting cluster will play the role of an ITS named "its1".
```

Comment:
s/Tell that the hosting cluster will play the role of an ITS named "its1"./Assign the hosting cluster the role of an ITS, named 'its1'."/

Reply:
Done.